src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -1062,6 +1062,10 @@ def _tulsi_sources_aspect(target, ctx):
         # else:
         #     module_name = None
 
+    if not bundle_id:
```

Comment:
Minor: Consider moving this around the above lines ( https://github.com/bazel-ios/tulsi/pull/3/files#diff-8b7a1c51d8820e04ad285e847917428444ea9ef42ddf070ef0cadc46421e7539R969 ) there's already some code to handle setting `bundle_name` and `bundle_id`.